components/react/hooks/src/useQuery/index.js

```diff
@@ -0,0 +1,42 @@
+import {useEffect, useRef} from 'react'
+
+import useRequest from '../useRequest/index.js'
+export * from '../useRequest/index.js'
+
+const useQuery = (query, {refetchInterval, isExecuteOnMountDisabled, ...options} = {}, deps = []) => {
+  const {data, refetch, ...others} = useRequest(query, options)
```

Comment:
This request is triggered even when the `isExecuteOnMountDisabled` is true. It would be nice to disable the request completely for those cases where you have nested requests to be done.